sdk/java-sdk/src/main/scala/kalix/javasdk/KalixRunner.scala

```diff
@@ -51,13 +52,18 @@ import org.slf4j.LoggerFactory
 object KalixRunner {
   object BindFailure extends Reason
 
-  final case class Configuration(userFunctionInterface: String, userFunctionPort: Int, snapshotEvery: Int) {
+  final case class Configuration(
```

Comment:
I guess we don't need to keep source compatibility for this class even though it's public/